Sources/NIO/ByteBuffer-views.swift

```diff
@@ -66,6 +66,14 @@ public struct ByteBufferView: RandomAccessCollection {
             return try body(bytes.bindMemory(to: UInt8.self))
         }
     }
+
+    /// Returns a `ByteBuffer` slice from the same range as the view.
+    ///
+    /// - returns: A `ByteBuffer` slice from the same range as the view or `nil` if the bytes are not readable in the
+    /// initial `ByteBuffer`.
+    public func getByteBuffer() -> ByteBuffer? {
```

Comment:
I don't think this needs to return `nil`, as it is not possible to obtain a `ByteBufferView` of non-readable bytes anyway. I think we can safely just force-unwrap the return value of `getSlice`.

Reply:
Thanks for the suggestion. I changed the method to return a non-optional `ByteBuffer` instead.